var/spack/repos/builtin/packages/r-countrycode/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RCountrycode(RPackage):
+    """Countrycode standardizes country names, converts them into
+    ~40 different coding schemes, and assigns region descriptors."""
+
+    homepage = "https://vincentarelbundock.github.io/countrycode/"
+    url      = "https://cran.r-project.org/src/contrib/countrycode_1.2.0.tar.gz"
```

Comment:
```suggestion
    url      = "https://cloud.r-project.org/src/contrib/countrycode_1.2.0.tar.gz"
    list_url = "https://cloud.r-project.org/src/contrib/Archive/countrycode"
```
This will download from the CRAN mirror and work even when a newer release comes out and 1.2.0 gets pushed to the archive.